### PR TITLE
Imports the trayApp_handle.py in Darwin

### PR DESCRIPTION
**Pull request by Avenred**
* Imports trayApp_handle in darwin.py instead of copy and pasting code.
  * Unfortunately loading the config file is not a function yet, so some copy pasting is required.

### CODE REVIEW

**Comment by Avenred** [comment]
I'm not sure if this code works. @LueWasHere please review.

**Comment by LueWasHere** [comment]
Reviewed.

File: capscontrol/trayApp_handle.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def main_loop(icon):
                 print("Caps off")
                 keyboard.press(Key.caps_lock)  # why the fuck is this an integer?
                 keyboard.release(Key.caps_lock)  # @LueWasHere help help help.
+                # the reason it is an int is because its a hex address (either that or it's the key code: https://wiki.osdev.org/PS/2_Keyboard#Scan_Code_Set_1 )
```
Comment (Avenred) [inline capscontrol/trayApp_handle.py:74]: This still reports as an error in my IDE... I'll just take your word for it.

**Review by Avenred — COMMENTED** [review]
Whoops I meant to leave a comment, not start a review!!!

**Pull request merged**